Enigmatry.Entry.Blueprint.Api.Tests/Infrastructure/ApiWebApplicationFactory.cs

```diff
@@ -16,18 +17,28 @@
 
 namespace Enigmatry.Entry.Blueprint.Api.Tests.Infrastructure;
 
-internal class ApiWebApplicationFactory(IConfiguration configuration, bool isUserAuthenticated = true) : WebApplicationFactory<Program>
+internal class ApiWebApplicationFactory : WebApplicationFactory<Program>
 {
-    protected override void ConfigureWebHost(IWebHostBuilder builder)
+    private readonly IConfiguration _configuration;
+    private readonly bool _isUserAuthenticated;
+
+    public ApiWebApplicationFactory(IConfiguration configuration, bool isUserAuthenticated = true)
     {
+        _configuration = configuration;
+        _isUserAuthenticated = isUserAuthenticated;
+        
+        // needed so that log statements from the Api appear in the test console runner
+        Server.PreserveExecutionContext = true;
```

Comment:
This is needed for the log statements of the production code to appear in the tests.